platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/PlatformTestingToolState.java

```diff
@@ -243,8 +244,16 @@ public class PlatformTestingToolState extends PartialNaryMerkleInternal implemen
 
     private long transactionsIgnoredByExpectedMap = 0;
 
-    public PlatformTestingToolState() {
-        super(ChildIndices.CHILD_COUNT);
+    public PlatformTestingToolState(
+            @NonNull final MerkleStateLifecycles lifecycles,
+            @NonNull final Function<SemanticVersion, SoftwareVersion> versionFactory) {
+        super(lifecycles, versionFactory);
+
+        //        logger.info(
```

Comment:
There's a lot of commented-out code in this class. Please either remove it, or uncomment it.

Reply:
I cleaned up `PlatformTestingToolState`